source/common/http/codec_client.cc

```diff
@@ -117,6 +117,9 @@ void CodecClient::onEvent(Network::ConnectionEvent event) {
               StreamInfo::ResponseFlag::UpstreamProtocolError);
         }
       }
+    } else {
+      ENVOY_CONN_LOG(warn, "Connection is closed by {} during connecting.", *connection_,
```

Comment:
should this be at warning level or can we downgrade to info / trace?

Reply:
This means connection wasn't even able to send CHLO. I think it's alarming enough for warning log, but if it could cause log spam, downgrading sounds reasonable.